.github/workflows/CI.yml

```diff
@@ -9,19 +9,19 @@ concurrency:
   # Skip intermediate builds: always.
   # Cancel intermediate builds: only if it is a pull request build.
   group: ${{ github.workflow }}-${{ github.ref }}
-  cancel-in-progress: ${{ startsWith(github.ref, 'refs/pull/') }}
+  cancel-in-progress: false
 jobs:
   test:
     name: Julia CI
     runs-on: ${{ matrix.os }}
-    timeout-minutes: 60
+    timeout-minutes: 90
     continue-on-error: ${{ matrix.version == 'nightly' }}
     strategy:
       fail-fast: false
       matrix:
         version:
-          - '1'
           - '1.6'
+          - '1.7'
```

Comment:
Why not 1.8?

Reply:
Because I wanted to fix CI first

Reply:
Let's merge and switch to 1.8 in a separate PR?

Reply:
Deal!